pkg/clients/grpc/manager/manager.go

```diff
@@ -7,9 +7,20 @@ import (
 	"github.com/ultravioletrs/cocos/pkg/clients/grpc"
 )
 
+type ManagerConfig struct {
+	grpc.BaseConfig
+	BackendInfo string `env:"BACKEND_INFO"    envDefault:""`
```

Comment:
manager client config is not configured using backendinfo, this exclusive to agent client(CLI). also clienttls should be part of base client config